### ISSUE TITLE
Bug: ReplaceText does not work with regexMatchHandler

### ISSUE THREAD

**Issue by sebastianhaeni** [xceedsoftware/DocX#211]
When using `ReplaceText` with the parameter `matchFormatting` set to `null`, I would expect it to match and replace texts irregardless of formatting. The comment in the code reaffirms that assumption too. But there is no implementation of it.

When using the `ReplaceText` method that has no `regexMatchHandler`, it is implemented and works.

https://github.com/xceedsoftware/DocX/blob/8649a57586e4145d8634e0e04a1c93386489e94e/Xceed.Words.NET/Src/Paragraph.cs#L3439

The example `Document` only works because in the source `.docx` file, there is no formatting, thus it matches `new Formatting()`.

https://github.com/xceedsoftware/DocX/blob/80655a981912c1e032d4fc326f98cfa73d73b25b/Examples/Samples/Document/DocumentSample.cs#L69

**Comment by XceedBoucherS** [comment]
Hi,
This is already fixed in v1.4.
You can see it in the Release notes of v1.4 available here : https://xceed.com/release-notes/#words-net-1.4.17620.09420.

You can get the v1.4 release here : https://xceed.com/xceed-words-for-net/
Users of the "Free" version currently have v1.1.
The "Free" version is always some vereions behind.
Thank you.

**Comment by themightylc** [comment]
Bug is in Paragraph.cs. The actual replacement is inside the null-check of matchFormatting instead of after it.
TL;DR: The closing curly bracket in line 3613 belongs in line 3605 instead.
